src/Components/InputElement/InputElement.tsx

```diff
@@ -4,27 +4,30 @@ import { UserSignupPosting } from '../../interfaces';
 type checkedInputType = 'first-name' | 'last-name' | 'email' | 'password' | 'repeat-password';
 
 interface Props {
+  isLoading: boolean,
   typeInput: checkedInputType,
   user: UserSignupPosting,
-  setPassword: (repeatPassword: string) => void
-  setUser: (user: UserSignupPosting) => void
-  setError: (error: string) => void
-}
+  setPassword: (repeatPassword: string) => void,
+  setUser: (user: UserSignupPosting) => void,
+  setError: (error: string) => void;
+};
 
 enum checkType {
```

Comment:
Since we agreed on leaving the last key of the Props empty gotta an extra semicolon hanging. I can nab this one